pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -94,6 +94,35 @@ func (cs *ConntrackConnectionStore) Run(stopCh <-chan struct{}) {
 	}
 }
 
+func (cs *ConntrackConnectionStore) RunPeriodicDeletion(stopCh <-chan struct{}) {
```

Comment:
I haven't reviewed this in details yet, but I am trying to understand why this is not handled by `deleteIfStaleOrResetConn` in `Poll` already? And maybe it would be the right place to do it instead of introducing this new logic & goroutine?

Reply:
Yes, we could have handled it there as well but that will be applicable only for the conntrack connections. We cannot do it for denied connections.
I have chosen this approach to have consistent logic for both the conntrack and denied connections.

Reply:
After review PR #2360  that is refactoring this code, I moved the deletion logic for conntrack connection to the existing goroutine. Te referenced PR has a goroutine to delete the deny connections, the stale connection logic can be added there when it is merged.